static/src/javascripts/projects/commercial/modules/dfp/on-slot-load.js

```diff
@@ -20,20 +20,15 @@ const onLoad = (event: SlotOnloadEvent) => {
     if (!advert) {
         return;
     }
-    if (
-        advert.size &&
-        ((typeof advert.size === 'string' && advert.size === 'fluid') ||
-            (advert.size[0] === 0 && advert.size[1] === 0))
-    ) {
-        const iframe = advert.node.getElementsByTagName('iframe')[0];
-        postMessage(
-            {
-                id: iframe.id,
-                host,
-            },
-            iframe.contentWindow
-        );
-    }
+
+    const iframe = advert.node.getElementsByTagName('iframe')[0];
```

Comment:
Will this throw an error if there is no element at [0]? Similarly if you try to call `.id` on it in this case?

Reply:
Yes it will...but that will never(!) happen. If the `advert` exists, then it is because it has been served by DFP, and this will always be an `iframe`.